python-bindings/overview_article/tasks.py

```diff
@@ -1,48 +1,95 @@
 """ Task definitions for invoke command line utility for python bindings
-    overview article. """
+    overview article.
+
+    If anything is causing trouble get in touch at
+    Znunu @ github or VimVim @ fcZBB2v @ discord"""
 import cffi
 import invoke
 import pathlib
+import sys
+import os
+import re
+
+on_win = sys.platform.startswith("win")
 
 
 @invoke.task
 def clean(c):
     """ Remove any built objects """
-    for pattern in ["*.o", "*.so", "cffi_example* cython_wrapper.cpp"]:
-        c.run("rm -rf {}".format(pattern))
+    for pattern in (
+        "*.o",
+        "*.so",
+        "*.obj",
+        "*.dll",
+        "*.exp",
+        "*.lib",
+        "*.pyx",
```

Comment:
Aha!  This is a problem.  I missed this one at first.  The *.pyx file is actually hand-build and need to NOT be removed.  

This causes the `invoke build-cython test-cython` sequence to fail.